react/store/createReturnRequest/components/OrderList.tsx

```diff
@@ -18,74 +18,94 @@ interface RowData {
+            day="2-digit"
+            month="2-digit"
+            year="numeric"
+          />
+        )
       },
-      status: {
-        title: (
-          <FormattedMessage id="store/return-app.return-order-list.table-header.items-to-return" />
-        ),
-        cellRenderer: function availableProducts({ rowData }: RowData) {
-          const { quantityAvailable, quantity } = createItemsSummary(rowData)
+      minWidth: 120,
+    },
+    status: {
+      title: (
+        <FormattedMessage id="store/return-app.return-order-list.table-header.items-to-return" />
+      ),
+      cellRenderer: function availableProducts({ rowData }: RowData) {
+        const { quantityAvailable, quantity } = createItemsSummary(rowData)
 
-          return <p>{`${quantityAvailable} / ${quantity}`}</p>
-        },
+        return <p>{`${quantityAvailable} / ${quantity}`}</p>
       },
-      selectOrder: {
-        title: (
-          <FormattedMessage id="store/return-app.return-order-list.table-header.select-order" />
-        ),
-        cellRenderer: function SelectOrderButton({ rowData }: RowData) {
-          const { quantityAvailable } = createItemsSummary(rowData)
+    },
+    selectOrder: {
+      title: (
+        <FormattedMessage id="store/return-app.return-order-list.table-header.select-order" />
+      ),
+      cellRenderer: function SelectOrderButton({ rowData }: RowData) {
+        const { quantityAvailable } = createItemsSummary(rowData)
 
-          return (
-            <div>
-              <Button
-                {...(!quantityAvailable
-                  ? null
-                  : {
-                      onClick: () =>
-                        navigate({
-                          to: `#/my-returns/add/${rowData.orderId}`,
-                        }),
-                    })}
-                variation="tertiary"
-                collapseLeft
-                disabled={!quantityAvailable}
-              >
-                <FormattedMessage id="store/return-app.return-order-list.table-header.select-order" />
-              </Button>
-            </div>
-          )
-        },
+        return (
+          <div>
+            <Button
+              {...(!quantityAvailable
+                ? null
+                : {
+                    onClick: () =>
+                      navigate({
+                        to: `#/my-returns/add/${rowData.orderId}`,
+                      }),
+                  })}
+              variation="tertiary"
+              collapseLeft
+              disabled={!quantityAvailable}
+            >
+              <FormattedMessage id="store/return-app.return-order-list.table-header.select-order" />
+            </Button>
+          </div>
+        )
       },
+      minWidth: 150,
     },
   }
+
+  const mobileOrder = {
+    orderId: null,
+    selectOrder: null,
+  }
+
+  return {
+    properties: isSmallScreen
+      ? Object.assign(mobileOrder, properties)
+      : properties,
+  }
```

Comment:
Object assign takes all enumerable properties from the second object and attaches them to the 1st one. As such, it will maintain the order and fill the values, this allows us to 'order' the schema columns